src/README.md

```diff
@@ -0,0 +1,27 @@
+# Run Tribler
+
+Full instruction: [install requirements](https://github.com/Tribler/tribler#setting-up-your-development-environment)
+
+```
+python3 -m pip install -r requirements.txt
+
+tribler.sh
+```
+
+# Run Tests
+
+Install all necessary dependencies:
+
+```
+python3 -m pip install -r requirements-dev.txt
+```
+
+Run tests:
+
+```
+export PYTHONPATH=${PYTHONPATH}:`echo {pyipv8,tribler-common,tribler-core,tribler-gui} | tr " " :`
+
+python3 -m pytest tribler-core
+python3 -m pytest tribler-common
+python3 -m pytest tribler-gui
```

Comment:
Running the GUI tests requires the `--guitests` flag, since they are heavy and spawn separate processes.

Reply:
fixed